parsers/powershell.c

```diff
@@ -49,6 +50,7 @@ static kindDefinition PowerShellKinds[COUNT_KIND] = {
 	{ true, 'v', "variable",	"variables" },
 	{ true, 'c', "class",		"classes" },
 	{ true, 'i', "filter",		"filter" },
+	{ true, 'e', "enum",		"enums" },
```

Comment:
`g` can be another candidate for `enum`.

You may want to extract items inside enum like `Label11` and `Label12`.
If that case, what kind name will you give to the items?

I don't know the terms of PowerShell. However, we can find hints in C parser:

```
$ ./ctags --list-kinds-full=C
#LETTER NAME       ENABLED REFONLY NROLES MASTER DESCRIPTION
D       macroparam no      no      0      C      parameters inside macro definitions
L       label      no      no      0      C      goto labels
Y       unknown    no      yes     1      C      unknown identifier
d       macro      yes     no      2      C      macro definitions
e       enumerator yes     no      0      C      enumerators (values inside an enumeration)
f       function   yes     no      0      C      function definitions
g       enum       yes     no      0      C      enumeration names
...
```

The C parser calls items inside an enum "enumerator".
For enum, the C parser uses `g` as its kind letter.
For enumerator, the C parser uses `e` as its kind letter.

You can make the PowerShell parser similar to the C parser.
Of course, the parsers in ctags are independent each other, So what I wrote here is nothing more than a comment.

Reply:
LETTER for kinds in PowerShell What should we do?

As for ctags, I think it is better to make them in a form similar to the C and C++ languages.

Since my use of ctags is not so deep, I have not come up with a good way to choose a LETTER.
In such a case, I think it is better to choose the LETTER in conjunction with its neighbors.
In view of ctags, PowerShell is similar to C and C++ languages, so I think it is better to make it similar to the C parser.

For the kind for the enum name, let us pick `g` to match the C language.

In the future, when choosing labels for enums and properties for classes, the C parser may be helpful as well.

Reply:
Good.